cypress/commons/actions/generic/ScenarioParameters.js

```diff
@@ -39,27 +36,10 @@ function getParametersConfirmDiscardButton() {
 function getLaunchButton(timeout) {
   return cy.get(GENERIC_SELECTORS.scenario.parameters.launchButton, timeout ? { timeout: timeout * 1000 } : undefined);
 }
-function getParametersUpdateAndLaunchButton(timeout) {
-  return cy.get(
-    GENERIC_SELECTORS.scenario.parameters.updateAndLaunchButton,
-    timeout ? { timeout: timeout * 1000 } : undefined
-  );
-}
-function getLaunchConfirmDialog() {
-  return cy.get(GENERIC_SELECTORS.scenario.parameters.dialogLaunch.dialogTitle);
-}
-function getLaunchConfirmButton() {
-  return cy.get(GENERIC_SELECTORS.scenario.parameters.dialogLaunch.confirmButton);
-}
-function getLaunchCancelButton() {
-  return cy.get(GENERIC_SELECTORS.scenario.parameters.dialogLaunch.cancelButton);
-}
-function getDontAskAgainCheckbox() {
-  return cy.get(GENERIC_SELECTORS.scenario.parameters.dialogLaunch.dontAskAgainCheckbox).find('input');
-}
-function checkDontAskAgain() {
-  getDontAskAgainCheckbox().check();
+function getSaveButton(timeout) {
+  return cy.get(GENERIC_SELECTORS.scenario.parameters.saveButton, timeout);
```

Comment:
Watch the type of the `timeout` parameter. To be as simple as possible, and to be consistent with other functions, we probably want it to be a **number of seconds**, so you need to use `timeout ? { timeout: timeout * 1000 }` (or just `{ timeout: timeout * 1000 }`) instead of only `timeout` when calling `cy.get`.

Also, if we never need to have a specific timeout value when getting the save button, you can remove the `timeout` parameter